ld-relay.go

```diff
@@ -45,10 +45,13 @@ type Config struct {
 	Environment map[string]*EnvConfig
 }
 
+type StatusEntry struct {
+	Status string `json:"status"`
+}
+
 var configFile string
 
 func main() {
```

Comment:
What will happen if a connection comes in for an environment that is not yet initialized?